nengo_gui/static/netgraph.js

```diff
@@ -335,7 +335,7 @@ Nengo.NetGraph.prototype.on_message = function(event) {
         var uid = data.uid;
         for (var i = 0; i < Nengo.Component.components.length; i++) {
             if (Nengo.Component.components[i].uid === uid) {
-                Nengo.Component.components[i].remove(true);
+                Nengo.Component.components[i].remove(true, data.report_back);
```

Comment:
Yuck...   I usually prefer function parameters that indicate what the function is supposed to do, rather than function parameters that indicate the context in which someone else called the function.  If ```report_back``` isn't clear enough, I'd be more tempted to do something like ```indicate_client_should_report_removal_back_to_server```, but maybe not quite as silly as that.  Can you say more about why you find ```report_back``` to be ambiguous?  I'm having trouble seeing what's unclear about it.